paasta_tools/setup_marathon_job.py

```diff
@@ -657,6 +660,19 @@ def main():
 
 
 def deploy_marathon_service(service, instance, client, soa_dir, marathon_config, marathon_apps):
+    """deploy the service instance given and proccess return code
+    if there was an error we send a sensu alert.
+
+    :param service: The service name to setup
+    :param instance: The instance of the service to setup
+    :param client: A MarathonClient object
+    :param soa_dir: Path to yelpsoa configs
+    :param marathon_config: The service instance's configuration dict
+    :param marathon_apps: A list of all marathon app objects
+    :returns: A tuple of (status, bounce_in_seconds) to be used by paasta-deployd
```

Comment:
I did think I might at the time. We actually have two different tuples currently. I think I'll leave this be for now but I might "make this better" in my next stab at this file.